Sources/Utilities/Factories/TabbedViewSectionFactory.swift

```diff
@@ -42,14 +43,21 @@ struct TabbedViewSectionFactory {
         let signoutSection = createSection(header: nil,
                                            footer: nil,
                                            cellModels: [.init(cellTitle: "app_signOutButton",
-                                                              textColor: .gdsRed)])
+                                                              textColor: .gdsRed) {
+            action()
+        }])
 
         return [manageDetailsSection,
                 legalSection,
                 helpSection,
                 signoutSection]
     }
 
+    @MainActor func signOutPage(coordinator: ProfileCoordinator?) {
+        coordinator!.openSignOutPage()
```

Comment:
please change to `coordinator?.openSignOutPage()`

Reply:
I don't think this is actually being used anywhere at the moment, so I'll remove it - unless you think otherwise?

Reply:
Removing is fine if unused